### PR TITLE
Add docker to run tests easily

### PR DESCRIPTION
**Pull request by leonardocustodio**
Just run `docker compose up` in the root folder.
The `polkadart_example` is not required yet to run the tests. But I'm leaving in there anyway for the next milestone.

### CODE REVIEW

**Review by justkawal — APPROVED** [review]
LGTM

**Comment by gabrielokura** [comment]
I'm getting this error when i run `docker compose up`
```
 => [ 2/12] RUN apt-get update                                                                                                                                                                                                                                                                                   7.2s
 => ERROR [ 3/12] RUN apt-get install -y curl git wget unzip libgconf-2-4 gdb libstdc++6 libglu1-mesa fonts-droid-fallback lib32stdc++6 python3                                                                                                                                                                  1.1s
------
 > [ 3/12] RUN apt-get install -y curl git wget unzip libgconf-2-4 gdb libstdc++6 libglu1-mesa fonts-droid-fallback lib32stdc++6 python3:
#0 0.343 Reading package lists...
#0 0.902 Building dependency tree...
#0 1.032 Reading state information...
#0 1.080 E: Unable to locate package lib32stdc++6
#0 1.080 E: Couldn't find any package by regex 'lib32stdc++6'
------
failed to solve: executor failed running [/bin/sh -c apt-get install -y curl git wget unzip libgconf-2-4 gdb libstdc++6 libglu1-mesa fonts-droid-fallback lib32stdc++6 python3]: exit code: 100
```

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:22.04
+
+RUN apt-get update
+RUN apt-get install -y curl git wget unzip libgconf-2-4 gdb libstdc++6 libglu1-mesa fonts-droid-fallback lib32stdc++6 python3
```
Comment (justkawal) [inline docker/Dockerfile:4]:
```suggestion
RUN apt-get install -y curl git wget unzip libglu1-mesa
```

**Review by justkawal — CHANGES_REQUESTED** [review]
With the suggested change the issue is gone.

**Review by justkawal — APPROVED** [review]
LGTM

**Review by Lohann — APPROVED** [review]

**Pull request merged**